spotify_player/src/state/ui/page.rs

```diff
@@ -89,7 +89,7 @@ pub enum ContextPageUIState {
 
 #[derive(Copy, Clone, Debug, PartialEq, Eq)]
 pub enum LibraryFocusState {
-    Playlists,
+    Playlists(i32),
```

Comment:
what is the inner value and why is it needed? This variant should represent the `Playlists` window and IIUC, there can only be one `playlists` window